priv/repo/migrations/20230320111909_drop_group.exs

```diff
@@ -0,0 +1,7 @@
+defmodule Dbservice.Repo.Migrations.DropGroup do
+  use Ecto.Migration
+
+  def change do
+    drop table("group")
```

Comment:
is there data on prod that will be affected by this?

Reply:
yes, the data will be affected by this. We need to re-sync the data again in this case